app/src/main/java/swati4star/createpdf/preferences/TextToPdfDefaultManager.java

```diff
@@ -0,0 +1,76 @@
+package swati4star.createpdf.preferences;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.support.annotation.NonNull;
+
+import swati4star.createpdf.util.Constants;
+
+import static swati4star.createpdf.util.Constants.DEFAULT_PAGE_COLOR;
+
+/**
+ * The {@link TextToPdfDefaultManager} is responsible for managing the default enhancement values
+ * for the Text-to-PDF enhancements.
+ */
+public class TextToPdfDefaultManager {
+
+    private final SharedPreferences mSharedPreferences;
+
+    public TextToPdfDefaultManager(@NonNull final Context context) {
+        mSharedPreferences = PreferenceManager.getDefaultSharedPreferences(context);
+    }
+
+    public int getFontColor() {
+        return mSharedPreferences.getInt(Constants.DEFAULT_FONT_COLOR_TEXT,
+                Constants.DEFAULT_FONT_COLOR);
+    }
+
+    public void updateFontColor(final int fontColor) {
```

Comment:
add javadoc style comments to all the newly added functions in this file

Reply:
Sounds good. I'll also change the method names to 'setXYZ' instead of 'updateXYZ' since that is a more used name for setters.

Reply:
@Swati4star Updated in latest.